test/e2e/lib/npd/npd.go

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package npd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path"
+	"strings"
+	"time"
+
+	"k8s.io/node-problem-detector/pkg/util/metrics"
+	"k8s.io/node-problem-detector/test/e2e/lib/gce"
+
+	"github.com/avast/retry-go"
+)
+
+const npdMetricsFilename = "npd_metrics.txt"
+const npdLogsFilename = "npd_logs.txt"
```

Comment:
Nit: Rename this to `node-problem-detector.log` so that it is consistent with k8s tests.

Reply:
Done.